src/include/parser/explain_statement.h

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// explain_statement.h
+//
+// Identification: src/include/parser/explain_statement.h
+//
+// Copyright (c) 2018, Pranav Kant
```

Comment:
Fix copyright.

Reply:
Okay. I will change it to "Carnegie Mellon University Database Group" but note that I am not part of the group. So writing that seemed wrong to me.